melodic/distribution.yaml

```diff
@@ -3766,6 +3766,29 @@ repositories:
       url: https://github.com/tu-darmstadt-ros-pkg/hector_slam.git
       version: melodic-devel
     status: maintained
+  heifu:
+    doc:
+      type: git
+      url: https://gitlab.pdmfc.com/drones/ros1/heifu.git
+      version: debian
+    release:
+      packages:
+      - heifu_bringup
+      - heifu_description
+      - heifu_diagnostic
+      - heifu_mavros
+      - heifu_msgs
+      - heifu_safety
+      - heifu_simple_waypoint
+      - heifu_tools
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/amferreiraBEV/heifu-release.git
```

Comment:
The `release` section lacks a `version` (see other entries in this file for an example).

Reply:
Should the version be the same as the tag name? Because it's "upstream/0.7.1".
Can it be 0.7.1?

Reply:
Looking at the latest Melodic release in this gbp repo I think it should be `0.7.1-2`.

Reply:
You're totally right! I'm still new at this things.

Reply:
Looking good now. No worry, next release is going to be much easier. :+1: